### PR TITLE
adding document that explains how reviews happen on PRs

### PR DESCRIPTION
**Pull request by arschles**
Fixes https://github.com/gomods/athens/issues/467

### CODE REVIEW

**Comment by kteb** [comment]
Maybe PR should be open for a certain time before merging (24H or 48H exluding week-ends ?).

My first PR: #457 was kind of a lot of things to avoid (I did a rebase after fixing the first issues) but it shows some interesting facts as I was slow to fix the the small catches (I'm on holidays so I do what I can :laughing:), some people asked me to change staff that looks mine but was actually not, and already merged. Also some good feedback came 1 or 2 days after the initial PR (which was on a weekend). So I think keeping them open for a while should be a requirement (the project was built so fast that people don't master everything yet).

**Comment by michalpristas** [comment]
@kteb we discussed this at the mtg a few weeks ago and in fact agreed on keeping important changes opened for more than a day. 
Quick fixes and small refactors can go in immediately as soon as some approver (besides yourself) approves it I guess.

**Comment by kteb** [comment]
@michalpristas ok you know it better than me, but I had the feeling that even changes that looks small go get better feedbacks from different reviewers.

File: REVIEWS.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+- If you're reviewing, it can help you understand what to do
+- If you're submitting a pull request, it can help you understand what folks
+are going to do, how to help them, and how to get your pull request
+merged faster
+
+## Steps
+
+We follow a lightweight list of steps in order to review 
+[pull requests in the Athens project](https://github.com/gomods/athens)
```
Comment (robjloranger) [inline REVIEWS.md]: should this be `https://github.com/gomods/athens/pulls`?

File: REVIEWS.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+(if you're not familiar with pull requests, see 
+[here](https://help.github.com/articles/about-pull-requests/)).
+
+Here they are:
+
+- Pull requests should be reviewed by at least one 
+[contributor](https://github.com/orgs/gomods/teams/contributors)
```
Comment (robjloranger) [inline REVIEWS.md]: currently this setting requires at least one review from members with write access, contributors only have read access.

File: REVIEWS.md (hunk 1 above)
Comment (carolynvs) [reply REVIEWS.md]: yeah I think that's more helpful to link to

**Comment by carolynvs** [comment]
FYI, I'm helping Aaron out this week and am finishing up some of his WIP PRs in time for GopherCon. 

I've addressed some of the feedback, but wasn't sure what we've agreed upon for the "how long to keep PRs open" question. Are we ok with merging this and then discussing that at a later dev meeting to come to an agreement?

**Comment by michalpristas** [comment]
@carolynvs we discussed that 24h or 36h should be enough, but feel free to leave out this information.

**Comment by carolynvs** [comment]
Cool, I'll include that as well.

File: REVIEWS.md (hunk 2 above)
Comment (carolynvs) [reply REVIEWS.md]: Fixed

**Comment by carolynvs** [comment]
@robjloranger Would you please take another look? I have fixed the doc to address your feedback.

**Pull request merged**